src/main/webapp/js/components/csv/csv-config-page.jsx

```diff
@@ -0,0 +1,252 @@
+          initialValues={this.props.csvConfig}
+          validate={this.validate}
+          mutators={{
+            ...arrayMutators,
+          }}
+          render={({
+            handleSubmit, values, invalid,
+          }) => (
+            <form onSubmit={handleSubmit}>
+              <TextField
+                key="name"
+                placeholder="Name"
+                disabled={!this.props.isNew}
+                fieldConfig={{
+                  name: 'name', displayName: 'CSV Name', required: true,
+                }}
+              />
+              <FieldArray name="csvFields">
+                { ({ fields }) => (
+                  <div>
+                    {fields.map((name, index) => (
+                      <div key={name} className="csvFields">
+                        {_.map(this.getFields(fields.value[index]), elem => renderFormField({ ...elem, name: `${name}.${elem.name}` }))}
+                        {(() => {
+                          // eslint-disable-next-line max-len
+                          const fieldConfig = this.props.fieldConfig[fields.value[index].fieldConfigId];
+                          if (fieldConfig && fieldConfig.fieldType === RELATION) {
+                            return (
+                              <div>
+                                <FieldArray name={`${name}.fieldValueMap`}>
+                                  { ({ fields: csvValueMapFields }) => (
+                                    <div>
+                                      {csvValueMapFields.map(csvRelatedField => (
+                                        <div key={csvRelatedField} className="csvRelatedField">
+                                          { _.map(this.getCsvValueMapFields(fieldConfig.relatedField, fieldConfig.relatedEntity), relatedElem => renderFormField({ ...relatedElem, name: `${csvRelatedField}.${relatedElem.name}` })) }
+                                          <button
+                                            type="button"
+                                            className="btn btn-danger"
+                                            onClick={() => csvValueMapFields.remove(index)}
+                                          >
+                                            Delete
+                                          </button>
+                                        </div>
+                                      ))}
+                                      <div>
+                                        <button
+                                          type="button"
+                                          className="btn btn-success my-2"
+                                          onClick={() => csvValueMapFields.push({
+                                            entityId: '',
+                                            fieldValue: '',
+                                          })}
+                                        >
+                                            Add related field
```

Comment:
you're not adding a related fields here, change it to "Add value mapping", "Add CSV value" or something similar